config/locales/en/loans.en.yml

```diff
@@ -18,6 +18,7 @@ en:
     delete: "Delete Loan"
     delete_responses: "Delete Responses"
     delete_selected: "Delete"
+    dependent_steps_notice: "Some of the selected steps depend on other steps. Their scheduled start dates cannot be adjusted directly."
```

Comment:
Actually... if the warning is showing, the operation will still work, right? Why do we need to show a warning at all? Won't the end result be that the date shows as updated?

I guess this only won't work if the depended-upon step is _not_ in the set of steps being adjusted. Should we test for that? I think it should be possible...

If we do, perhaps this should read "Some of the selected steps depend on other steps that are not selected. You can proceed, but the start dates of those steps will not be adjusted."